src/main.jsx

```diff
@@ -48,22 +50,18 @@ const router = createBrowserRouter([
     path: "/Login",
```

Comment:
lets have a talk about category path to be as close as possible to rest convention